src/index.html

```diff
@@ -9,6 +9,40 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="logo" href="#home">
+        <img class="logo__img"
+             src="./images/logo.png"
+             alt="Moyo">
```

Comment:
If the HTML-element has long attribute values or number of attributes is more than 2 - start each one, including the first, on the new line with 2-space indentation related to tag. Tag’s closing bracket should be on the same level as opening one.
```suggestion
        <img 
          class="logo__img"
          src="./images/logo.png"
          alt="Moyo"
        >
```